MTM-48188: announcement of nimbus-jwt usage in sso (#111)
SoftwareAG/c8y-releasenotes: content/release-10-17-0/announcements-10-17-0.md

#### Implemented

##### Support for new token type in single sign-on authentication

> Cumulocity IoT supports new standard OAUTH2 tokens with the "at+jwt" type header parameter. For more information on this parameter, see [https://datatracker.ietf.org/doc/html/rfc9068#name-header](https://datatracker.ietf.org/doc/html/rfc9068#name-header).

Reviewer: Let us call it OAuth2 access tokens.

> Single-sign-on users managed by new authorization servers (that is, generating a JWT token with type "at+jwt") can thus login successfully into Cumulocity IoT.

Reviewer: Same as above. Let us call it OAuth2 access tokens.

#### Implemented

> ##### Support for new token type "at+jwt" in SSO

Reviewer 2: ```suggestion
##### Support for new token type in single sign-on authentication
```

> Cumulocity supports new standard OAUTH2 tokens with "at+jwt" type Header Parameter. For more information, see https://datatracker.ietf.org/doc/html/rfc9068#name-header.

Reviewer 2: ```suggestion
Cumulocity IoT supports new standard OAUTH2 tokens with the "at+jwt" type header parameter. For more information on this parameter, see [https://datatracker.ietf.org/doc/html/rfc9068#name-header](https://datatracker.ietf.org/doc/html/rfc9068#name-header).
```

> Single-sign-on users managed by new authorization servers (i.e, generating JWT token with type "at+jwt") can thus login successfully into Cumulocity.

Reviewer 2: ```suggestion
Single-sign-on users managed by new authorization servers (that is, generating a JWT token with type "at+jwt") can thus login successfully into Cumulocity IoT.
```

> To process this new token type, Cumulocity uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt), which can be enabled by the tenants with the

Reviewer 2: ```suggestion
To process this new token type, Cumulocity IoT uses the [Nimbus JOSE + JWT](https://connect2id.com/products/nimbus-jose-jwt), which can be enabled by the tenants with the
```
